functions/controllers/user/index.js

```diff
@@ -3,14 +3,20 @@ const { signupUser } = require('./signUp');
 const { loginUser } = require('./login');
 const { imageUpload } = require('./imageUpload');
 const { getAllUser } = require('./getUsers');
+const { getAUser } = require('./getAUser');
+const { getAuthUserCredentials } = require('./getAuthUserCredentials');
 
 const login = (req, res) => loginUser(req, res, db);
 const signUp = (req, res) => signupUser(req, res, db);
 const image = (req, res) => imageUpload(req, res, db);
 const getAll = (req, res) => getAllUser(req, res, db);
+const getUserById = (req, res) => getAUser(req, res, db);
+const AuthUserCredentials = (req, res) => getAuthUserCredentials(req, res, db);
```

Comment:
this is a duplicate of the same function

Reply:
no it is not .. auth user gets the users private credentials

Reply:
@SapeleD3 The pattern that you are using from line 9-14 doesn't make much sense. Can you explain why you are creating these functions this way?

Reply:
i am breaking down the api to an MVC model making 9  - 14 the standard for each controllers minimizing code per function